src/main/resources/templates/home.html

```diff
@@ -4,9 +4,15 @@
 
 <head>
   <th:block th:insert="~{fragments/common :: head(title='')}"></th:block>
+  <link rel="stylesheet" th:href="@{'/css/welcome-message.css'}">
 </head>
 
 <body>
+<div id="welcome-message" class="welcome-message">
```

Comment:
Probably only want to show this for the exe version 
Anyone running docker should know what self hosting is

Reply:
Users in a company do not know what docker or self hosting is but use a company provided self hosted stirling instance (we do!). This feature should also probably be optional, not everybody likes first time popups

Reply:
Agreed I wouldn't want to show this to anyone outside of windows exe users
That's true for this whole feature 

As it is I believe this is AI generated PR and will be removed !